operator/pkg/manifest/operations.go

```diff
@@ -29,15 +29,15 @@ const (
 type ChartInfo struct {
 	ChartPath    string
 	RepoName     string
-	Url          string
+	URL          string
 	ChartName    string
 	ReleaseName  string
 	ClientConfig map[string]interface{}
 	Overrides    map[string]interface{}
 }
 
 type DeployInfo struct {
-	Ctx            context.Context
+	Ctx            context.Context //nolint:containedctx
```

Comment:
after check the usage of this Ctx, I think it's easy to pass context as an additional param to the function where it been used, then we can avoid to use DeloyInfo to carry it.

Basically it's used in `manifestOperations.Install(deployInfo)` and `manifestOperations.Uninstall(deployInfo)`, which is inside `HandleCharts`, so just need to give `handlerFn` and extra context param.

Reply:
Valid point, I will add a TODO and do it an a follow-up PR. Since this will require even further changes.

Reply:
OK